.github/workflows/ubuntu.yml

```diff
@@ -42,5 +42,12 @@ jobs:
       env:
         GH_TOKEN: ${{ github.token }}
     - name: Run test
+      # Tests don't pass yet on truffleruby
+      if: >-
+        matrix.ruby != 'truffleruby-head'
       run: bundle exec rake
       continue-on-error: ${{ startsWith(matrix.ruby, 'jruby') }}
+    - name: Install gem
+      run: |
+        gem install pkg/*.gem
+        ruby -rstringio -e 'puts StringIO::VERSION'
```

Comment:
Just for confirmation: This outputs `3.0.1` (not `3.1.1`) on truffleruby-head. It's intentional, right?

https://github.com/ruby/stringio/actions/runs/7032124053/job/19135198739?pr=74#step:9:9

```text
3.0.1
```

Reply:
Yes, because it uses truffleruby's stringio.rb.